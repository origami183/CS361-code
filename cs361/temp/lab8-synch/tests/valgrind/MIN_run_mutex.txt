==1982172== Memcheck, a memory error detector
==1982172== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1982172== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1982172== Command: ../synch -m
==1982172== 
Running mutual exclusion test
CHILD EXIT: 100000000
CHILD EXIT: 100000000
Sum of results: 200000000
Value of shared: 0
==1982172== 
==1982172== HEAP SUMMARY:
==1982172==     in use at exit: 0 bytes in 0 blocks
==1982172==   total heap usage: 11 allocs, 11 frees, 12,854 bytes allocated
==1982172== 
==1982172== All heap blocks were freed -- no leaks are possible
==1982172== 
==1982172== For lists of detected and suppressed errors, rerun with: -s
==1982172== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
