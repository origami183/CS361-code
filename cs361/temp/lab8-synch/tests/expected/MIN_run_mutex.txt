Running mutual exclusion test
CHILD EXIT: 100000000
CHILD EXIT: 100000000
Sum of results: 200000000
Value of shared: 0
